app/assets/components/lks-header/template.njk

```diff
@@ -1,162 +1,163 @@
-                </svg>
-                <span class="nhsuk-u-visually-hidden">Close search</span>
-              </button>
-            </form>
-          </div>
-        </div>
-
       </div>
-
-    </div>{# close nhsuk-header__container #}
-
-    <nav class="nhsuk-header__navigation" id="header-navigation" role="navigation" aria-label="Primary navigation" aria-labelledby="label-navigation">
-      <div class="nhsuk-width-container">
-        <p class="nhsuk-header__navigation-title"><span id="label-navigation">Menu</span>
-          <button class="nhsuk-header__navigation-close" id="close-menu">
-            <svg class="nhsuk-icon nhsuk-icon__close" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
-              <path d="M13.41 12l5.3-5.29a1 1 0 1 0-1.42-1.42L12 10.59l-5.29-5.3a1 1 0 0 0-1.42 1.42l5.3 5.29-5.3 5.29a1 1 0 0 0 0 1.42 1 1 0 0 0 1.42 0l5.29-5.3 5.29 5.3a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42z"></path>
-            </svg>
-            <span class="nhsuk-u-visually-hidden">Close menu</span>
-          </button>
-        </p>
-        <ul class="nhsuk-header__navigation-list">
-          <li class="nhsuk-header__navigation-item nhsuk-header__navigation-item--for-mobile">
-            <a class="nhsuk-header__navigation-link" href="{% if params.homeHref %}{{ params.homeHref }}{% else %}/{% endif %}">
-              {% if params.homeText %}{{ params.homeText }}{% else %}Home{% endif %}
-              <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
-                <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+      <div class="nhsuk-header__multi-logo-organisation-name">
+        <a
+          class="nhsuk-header__link"
+          href="/"
+          aria-label="Home"
+        >
+          <span class="nhsuk-organisation-name">
+            {{ params.organisation.name }}
+            {%- if params.organisation.split -%}
+              <span class="nhsuk-organisation-name-split">{{ params.organisation.split }}</span>
+            {%- endif -%}
+          </span>
+          {% if params.organisation.descriptor %}
+            <span class="nhsuk-organisation-descriptor">{{ params.organisation.descriptor }}</span>
+          {% endif %}
+          </a>
+      </div>
+    {%- else -%}
+      <div class="nhsuk-header__logo {%- if showNav == "false" and showSearch == "false" -%}nhsuk-header__logo--only{%- endif -%}">
+        {% if params.logos.fp %}
+          <a
+            class="nhsuk-header__link nhsuk-header__fp"
+            href="/"
+            aria-label="United Kingdom Foundation Programme Website (Opens in New Window)"
+            target="_blank"
+          >
+            <image src="/images/logos/ukfpo-logo.png" alt="Logo for United Kingdom Foundation Programme" ></image>
```

Comment:
Could we correct the image tag name (`<img>`) ?

Reply:
I could still see 6 instances of `<image>` tag usage on `app/assets/components/lks-header/template.njk`. Could we correct them as well ?